lib/Model.js

```diff
@@ -206,6 +206,45 @@ Model.compile = function compile (name, schema, options, base) {
     return table.buildTableReq(table.name, table.schema);
   };
 
+  function createTransactionFunction(func, val, optionsIndex, model) {
+    return async function () {
+      let args = [...arguments];
```

Comment:
Can't we just do:
```js
return async function (...args) {
  // ...
```

I see we use spread in other places too, and even have object spread, so I'd suspect this is also available to us.

Reply:
@jutaz Yes. Great idea! Will make that change soon.

Reply:
Just addressed this in latest commit